#hector_image_processing/crop    crop
crop_image_processing/Crop    crop
float64 endlap
float64 sidelap
float64 gsd
float64 frame_rate
float64 cam_ori
geometry_msgs/Vector3   ref_gps
geometry_msgs/Vector3   landing_gps
geometry_msgs/Vector3   uav_gps
geometry_msgs/Vector3   uav_Ori
float64 i_width
float64 i_height
float64 fx
float64 fy
float64 cx
float64 cy
float64 skew
float64 s_width
float64 s_heigth
float64 fl
float64 max_speed
bool are_degrees
---
bool    success
geometry_msgs/Vector3[] plan_gps_coors
geometry_msgs/Vector3[] crop_gps_coors
float64[] uav_heading
#hector_image_processing/image_point[] im_coors
float64 crop_area
float64 uav_speed
float64 trig_interval
float64 flight_height
